### DISCUSSION TITLE
''Community''

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1532] [Q&A]
Is SHVDN still ''community,'' and if yes, is it necessary to be?

**Comment by kagikn** [comment]
SHVDN is still "community" since I consider opinions given by others. I would definitely remove the word if I gave up considering them (e.g. not merging PRs), but I don't think I need to remove the word right now.

**Reply by Michael21107** [reply]
It sure is, I just don't think it's worth confusing people by having that word in only 1 place (afaik) - the title of the readme.

**Reply by kagikn** [reply]
Fair enough, will remove the word from the title of the readme.

**Reply by Michael21107** [reply]
With all the love, I'm saying this: Isn't the official name of SHVDN ScriptHookVDotNet?

**Reply by kagikn** [reply]
I would say both ScriptHookVDotNet and Script Hook V .NET are the official product names, and I can't determine which should be the official name actually. I don't care about whether the "Community" prefix is used much, but I wouldn't bother calling SHVDN Community Script Hook V .NET tbh. Idk about if the original owner crosire cared about the "Community" prefix, though.

**Reply by Michael21107** [reply]
I think it was just a little subtitle spread by being in the README and whatnot, and by newbies.

I would drop the Community for sure, and if you aren't certain about using only DotNet, I'd do a poll.

**Comment by WithLithum** [comment]
I think primarily that SHVDN was called "Community" because it was the "unofficial" continuation of the Script Hook Dot Net project which was for GTA IV.
